website/docs/r/record_set.html.md

```diff
@@ -43,11 +43,12 @@ The following arguments are supported:
 * `ttl` - (Optional) The DNS record set's TTL, or time to live.
 
 * `record_addresses` - (Optional) A list of the record set's addresses.
-  See [record addresses](#record-addresses) below for details.
 
-* `record_cname` - (Optional) If the record is a CNAME, the record's value.
+* `record_texts` - (Optional) If the record is a TXT record, a list of the record set's text values.
```

Comment:
We should also add `ownerGroupId` to this since it already has been added to the go resource file for record set.

Reply:
@mitruly Thanks! Good catch - I've pushed a fix for this.